### PR TITLE
Add custom error messages

### PR DESCRIPTION
**Pull request by gclssvglx**
- Add custom error messages from NSV tool
- Add custom validations for `:need_help_with` and `:nation`

:warning: This application is Continuously Deployed: :warning:

- Merged changes are automatically deployed to staging and production.

- Make sure you follow [the guidance for deployments](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) **before** you merge.

- Check your branch is being deployed in the [Release app](https://release.publishing.service.gov.uk/applications/smartanswers), after merging.

[Trello](https://trello.com/c/jGfbTFWY/447-add-error-messages-from-nsv-tool-into-smart-answers)

### CODE REVIEW

**Review by reggieb — APPROVED** [review]
LGTM - Some very minor comments that can be ignored.

File: lib/smart_answer_flows/coronavirus-find-support.rb [hunk 1]
```diff
@@ -25,6 +25,10 @@ def define
           calculator.need_help_with = response
         end
 
+        validate do
+          calculator.valid_need_help_with?
+        end
+
```
Comment (theseanything) [inline lib/smart_answer_flows/coronavirus-find-support.rb:31]: Is it possible to use the `none_option` here?

https://github.com/alphagov/smart-answers/blob/master/doc/smart-answers/question-types.md#checkbox_question

Comment (theseanything) [reply lib/smart_answer_flows/coronavirus-find-support.rb:31]: I.e.
```
 checkbox_question :need_help_with? do
        option :feeling_unsafe
        option :paying_bills
        option :getting_food
        option :being_unemployed
        option :going_to_work
        option :somewhere_to_live
        option :mental_health
        none_option
```

**Review by theseanything — CHANGES_REQUESTED** [review]

File: lib/smart_answer/calculators/coronavirus_find_support_calculator.rb [hunk 2]
```diff
@@ -15,6 +15,14 @@ class CoronavirusFindSupportCalculator
                   :have_you_been_made_unemployed,
                   :mental_health_worries
 
+    def valid_nation?
+      nation.present? && nation != "none"
```
[the remark this answers is not in the document]
Comment (gclssvglx) [reply lib/smart_answer/calculators/coronavirus_find_support_calculator.rb:19]: Thanks @reggieb - updated.

File: lib/smart_answer/calculators/coronavirus_find_support_calculator.rb [hunk 3]
```diff
@@ -15,6 +15,14 @@ class CoronavirusFindSupportCalculator
...
+    end
+
+    def valid_need_help_with?
+      need_help_with.present? && need_help_with != "none"
```
[the remark this answers is not in the document]
Comment (gclssvglx) [reply lib/smart_answer/calculators/coronavirus_find_support_calculator.rb:23]: Thanks @reggieb - updated.

File: lib/smart_answer_flows/coronavirus-find-support.rb (hunk 1 above)
Comment (gclssvglx) [reply lib/smart_answer_flows/coronavirus-find-support.rb]: Thanks @theseanything - I think this works well, and is possibly closer to the original intended behaviour.

File: lib/smart_answer_flows/coronavirus-find-support/questions/worried_about_work.erb [hunk 4]
```diff
@@ -11,3 +11,7 @@
   "no": "No",
   "not_sure": "Not sure",
 ) %>
+
+<% text_for :error_message do %>
+  Select yes if you’re worried about going in to work
+<% end %>
```
[the remark this answers is not in the document]
Comment (gclssvglx) [reply lib/smart_answer_flows/coronavirus-find-support/questions/worried_about_work.erb:17]: Thanks @theseanything - I feel this needs to be addressed as part of a different story.

**Review by theseanything — APPROVED** [review]

**Pull request merged**